modules/objdetect/include/opencv2/objdetect/face.hpp

```diff
@@ -57,7 +57,7 @@ class CV_EXPORTS_W FaceDetectorYN
     /** @brief A simple interface to detect face from given image
      *
      *  @param image an image to detect
-     *  @param faces detection results stored in a cv::Mat
+     *  @param faces detection results stored in a 2D cv::Mat of shape [num_faces, 15] (0-3: bbox top left corner and width and height, 4-13: landmark points starting from right eye to left eye, nose tip, right and left corners of mouth, 14: face score)
```

Comment:
I propose to add image with landmarks or detailed landmarks description or reference to the original project.